### ISSUE TITLE
FF57: doesn't work in embedded error pages

### ISSUE THREAD

**Issue by wavexx** [philc/vimium#2813]
I know this is a limitation in the FF web-extension, but I'd like to know if this issue has some upstream bug report that we could push for.

Keybindings do not work in embedded error pages and several built-in pages in FF such as the new tab page. For example, a page that failed to load cannot be handled with vimium keybindings anymore. I cannot overstate how frustrating this is, because it means that for several scenarios you have to use FF keybindings anyway, making the vimium/extension equivalent useless.

**Comment by Kristaba** [comment]
Indeed, this is extremely frustrating...
I can imagine possible security issues which could be the reason of this choice, but I also imagine/hope there is a possible compromise (such as a specific permission that should be granted to an extension to allow to work in those *privileged pages*).

As mentioned, all the equivalent extensions are facing this issue (such as [Saka key](https://github.com/lusakasa/saka-key/issues/94), [Vim Vixen](https://github.com/ueokande/vim-vixen/issues/208) and [Tridactyl](https://github.com/cmcaine/tridactyl/issues/113) ).
For what I know, @cmcaine is working on proposing an API to make that possible in Firefox (see [this issue on bugzilla](https://bugzilla.mozilla.org/show_bug.cgi?id=1215061) ).

I know that I am not legitimate to say that, as I am neither working on one of those plugins nor on Firefox. But it could be nice if some developers of all the concerned extensions were able to express their needs it terms of new API and check if the work of @cmcaine will cover them and/or work with Firefox to see what could be implemented.
I have the feeling that the major problem is that upstream is not always aware of the extension needs (especially when it is such *niche*-extension).

TL;DR: is it possible to work together with other extension developers to tackle those issue in WebExtension API as soon as possible?

**Comment by mrmr1993** [comment]
As far as I'm concerned, @cmcaine's keyboard API is not the right solution for us; if we don't have a content script, most of our commands are useless. A better solution would be to keep pressing the Firefox devs to inject content scripts into `about:`, `data:` (and, for this bug, error) pages.

Relevant Firefox bug: [[Bug 1269456] Add permission to allow extensions to load semi-privileged URLs](https://bugzilla.mozilla.org/show_bug.cgi?id=1269456).
